### ISSUE TITLE
Allow change of time limit of multiple test cases

### ISSUE THREAD

**Issue by lancehaoh** [dj3500/hightail#95]
Is it possible to allow users to change the time limit for multiple test cases at one time instead of manually clicking and changing each one?

**Comment by dj3500** [comment]
Yes, probably the time limit should be a property of a problem, not of a testcase. Then also newly added testcases would inherit the correct TL for the problem, instead of the standard 3s. Also see this comment on CF: http://codeforces.com/blog/entry/13141?#comment-406278
